src/azul/lambda_layer.py

```
@@ -41,18 +41,20 @@ def _update_required(self) -> bool:
             return False
 
     def update_layer(self, force: bool = False):
-        log.info('Using dependency layer package at s3://%s/%s.', config.layer_bucket, self.object_key)
+        log.info('Using dependency layer package at s3://%s/%s.', config.lambda_layer_bucket, self.object_key)
         if force or self._update_required():
             log.info('Staging layer package ...')
             input_zip = self.out_dir / 'deployment.zip'
             layer_zip = self.out_dir / 'layer.zip'
             if force:
                 log.info('Tainting current lambda layer resource to force update')
+                subprocess.run(['make', 'init'], cwd=Path(config.project_root) / 'terraform').check_returncode()
                 command = ['terraform', 'taint', 'aws_lambda_layer_version.dependencies']
-                subprocess.run(command, cwd=Path(config.project_root) / 'terraform')
+                if subprocess.run(command, cwd=Path(config.project_root) / 'terraform').returncode != 0:
```

See if `-allow-missing` can be used instead of ignoring all errors.